docs/source/user/input_file_overview.rst

```diff
@@ -0,0 +1,140 @@
+.. _input_file_overview:
+
+
+Overview of input file formats
+==============================
+
+OpenFAST uses two primary input file formats: *value column* where the first
+value on the line is read, and *key+value* where a value and keyword pair are
+read. Both formats are line number based where a specific input is expected on a
+specific line, with some exceptions. 
+
+.. _sec_value_column:
+
+Value column input files
+------------------------
+
+Only the first column in a *value column* based input file is read.  This is the
+historical format used by OpenFAST and it's predecessors. Everything after the
+first value read is simply ignored by the code.  This allowed the user to keep
+old values while modifying things.  So for example, and input line like
+
+::
+
+ 2       20   TMax            - Total run time (s)
+
+would be read as `2` and the `20` and everything after it ignored.
+
+This format and associated parsing methodology is somewhat limited in informing
+the user of errors in parsing, and limited the ability to pass entire inpute
+files as text strings from another code (such as a Python driver code).
+
+.. _sec_format_key_value:
+
+Key + Value input files
+-----------------------
+
+The first two columns are read in *key + value* input files.  One of these two
+columns must contain the **exact** keyword, and the other must contain the value
+that corresponds to it.  For example, an input line 
+
+::
+
+         20   TMax            - Total run time (s)
+
+is equivalent to
+
+::
+
+   TMax         20            - Total run time (s)
+
+One additional feature of this input file format is the ability to add an
+arbitrary number of comment lines wherever the user wishes.  Any line starting
+with `!`, `#`, or `%` will be treated as a comment line and ignored.  For
+example,
+
+
+::
+
+  ! This is a comment line that will be skipped
+          %  and this is also a comment line that will be skipped
+  # as is this comment line
+         20   TMax            - Total run time (s)
+  ! the first two columns in the above line will be read as the value + key
+
+The parser for this format of input file also tracks which lines were comments,
+and which lines contained the value and key pair.  If a keyname is not found the
+parser will return an error with information about which line it was reading
+from.
+
+
+Modules using Key + Value Format
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following modules use the *key + value* format input files (all other
+modules use the *value column* format):
```

Comment:
Add a note saying something like, "(note that *key + value* format and *value column* input files can be identical if the value is stated before the key)".